cocotb/triggers.py

```diff
@@ -161,11 +161,36 @@ def unprime(self):
 class Timer(GPITrigger):
     """Fires after the specified simulation time period has elapsed."""
     def __init__(self, time_ps, units=None):
+        """
+        .. Constructor. This RST comment works around sphinx-doc/sphinx#6885
```

Comment:
Don't think this is needed, it's only for when the docstring is inherited, and this one isn't

Reply:
Oh, right.